packages/wdio-config/tests/node/configparser.test.ts

```diff
@@ -1,5 +1,5 @@
 import url from 'node:url'
-import path from 'node:path'
+import path, { resolve } from 'node:path'
```

Comment:
I suggest to either import the default or named functions but we should not mix
```suggestion
import path from 'node:path'
```